client/polyfills.js

```diff
@@ -0,0 +1,24 @@
+import "core-js/es/map";
+import "core-js/es/set";
+import "core-js/es/array";
+import "core-js/es/promise";
+import "core-js/es/weak-map";
+import "core-js/es/string";
+import "regenerator-runtime/runtime";
+import "raf/polyfill";
+
+const removeToastsInOldBrowsers = () => {
+  setInterval(() => {
+    const toastContainer = document.querySelector(".Toastify").children;
+    if (toastContainer.length) {
+      const toasts = Array.from(toastContainer[0].children);
+      toasts.map((toast) => setTimeout(() => toast.remove(), 1000));
```

Comment:
I would let the toast to show for at least 4 seconds (the time to read it)

Reply:
Ok will change it.